bundles/org.openhab.voice.rustpotterks/src/main/resources/OH-INF/i18n/rustpotterks.properties

```diff
@@ -0,0 +1,37 @@
+voice.config.rustpotterks.averagedThreshold.label = Averaged Threshold
+voice.config.rustpotterks.averagedThreshold.description = Configures the detector averaged threshold, is the min score (in range 0. to 1.) that the averaged wakeword template should obtain to allow to continue with the detection. This way it can prevent to run the comparison of the current frame against each of the wakeword templates. If set to 0 this functionality is disabled.
+voice.config.rustpotterks.comparatorBandSize.label = Comparator Band Size
+voice.config.rustpotterks.comparatorBandSize.description = Configures the band-size for the comparator used to match the samples.
+voice.config.rustpotterks.comparatorRef.label = Comparator Ref
+voice.config.rustpotterks.comparatorRef.description = Configures the reference for the comparator used to match the samples.
+voice.config.rustpotterks.eagerMode.label = Eager Mode
+voice.config.rustpotterks.eagerMode.description = Enables eager mode. End detection as soon as a result is over the score, instead of waiting to see if the next frame has a higher score.
+voice.config.rustpotterks.group.vadDetector.label = Noise Detector
+voice.config.rustpotterks.group.vadDetector.description = Optional noise detection options.
```

Comment:
Please fix that by generating again the file.

And please add these line at the end of the file:
```

# service

service.voice.rustpotterks.label = Rustpotter Keyword Spotter
```